### PR TITLE
networkd: batman-adv parameter corrected (GatewayBandwidth)

### PR DESCRIPTION
**Pull request by joerg-d**

### CODE REVIEW

**Comment by bluca** [comment]
Please fix other references to the same properties, in:

test/fuzz/fuzz-netdev-parser/28-batadv.netdev
test/fuzz/fuzz-netdev-parser/directives.netdev
test/test-network/conf/25-batadv.netdev

File: src/network/netdev/netdev-gperf.gperf [hunk 1]
```diff
@@ -242,8 +242,8 @@ BatmanAdvanced.BridgeLoopAvoidance,       config_parse_bool,
 BatmanAdvanced.DistributedArpTable,       config_parse_bool,                         0,                             offsetof(BatmanAdvanced, distributed_arp_table)
 BatmanAdvanced.Fragmentation,             config_parse_bool,                         0,                             offsetof(BatmanAdvanced, fragmentation)
 BatmanAdvanced.GatewayMode,               config_parse_batadv_gateway_mode,          0,                             offsetof(BatmanAdvanced, gateway_mode)
-BatmanAdvanced.GatewayBandwithDown,       config_parse_badadv_bandwidth,             0,                             offsetof(BatmanAdvanced, gateway_bandwidth_down)
-BatmanAdvanced.GatewayBandwithUp,         config_parse_badadv_bandwidth,             0,                             offsetof(BatmanAdvanced, gateway_bandwidth_up)
```
Comment (yuwata) [inline src/network/netdev/netdev-gperf.gperf]: Ugh... Please keep these settings for backward compatibility...

Comment (bluca) [reply src/network/netdev/netdev-gperf.gperf]: did we ship v248 with it? thought it was new

Comment (yuwata) [reply src/network/netdev/netdev-gperf.gperf]: The settings are already shipped in v248... Unfortunately.

Comment (yuwata) [reply src/network/netdev/netdev-gperf.gperf]: Hm, but after re-configuring, I noticed that the wrong options are not documented. Let's merge this as is.

**Review by yuwata — APPROVED** [review]

**Comment by joerg-d** [comment]
I am not sure if my last commit to preserve the current options is the right way. If not, please someone who knows better about systemd will have to do it.

**Comment by yuwata** [comment]
LGTM. Thanks.

File: test/fuzz/fuzz-netdev-parser/directives.netdev [hunk 2]
```diff
@@ -232,6 +232,6 @@ DistributedArpTable=
 Fragmentation=
 HopPenalty=
 OriginatorIntervalSec=
-GatewayBandwithDown=
-GatewayBandwithUp=
```
Comment (yuwata) [inline test/fuzz/fuzz-netdev-parser/directives.netdev]: Oh, then you need to keep this two lines.

**Review by yuwata — CHANGES_REQUESTED** [review]

**Review by yuwata — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**